_posts/2020-03-15-method-s-arguments-pt-1.markdown

```diff
@@ -0,0 +1,373 @@
+---
+layout: post
+title: Ruby Method's Arguments
+date: 2020-03-15 09:30:00
+categories: [ruby, learning]
+author: arieljuod
+---
+
+Ruby is an object oriented language where everything is an object (even methods are objects of the class Method!), so everything we need to do is done by calling methods on objects. That also means that methods have to provide a lot of flexibility because they are used everywhere.
```

Comment:
@arielj maybe add a link to the method class?

Reply:
I had part1 duplicated here, I removed the file. This PR is only for part2 but I'll update this suggestions for part1 on another PR